src/AvatarStack/AvatarStack.tsx

```diff
@@ -4,22 +4,53 @@ import styled from 'styled-components'
 import {get} from '../constants'
 import Box from '../Box'
 import sx, {SxProp} from '../sx'
+import {AvatarProps, DEFAULT_AVATAR_SIZE} from '../Avatar/Avatar'
 
 type StyledAvatarStackWrapperProps = {
   count?: number
 } & SxProp
 
+const findSmallestNumber = (numbers: number[]): number => {
```

Comment:
Would `Math.min(...numbers)` work here? 🤔  Just an idea - not a blocker at all.

Reply:
Great suggestion! Soooo much easier.